tests/lib/rules/jsx-sort-props.js

```diff
@@ -269,8 +269,21 @@ ruleTester.run('jsx-sort-props', rule, {
       errors: [expectedError]
     },
     {
+      code: '<App key={2} b a />',
+      options: reservedFirstAsBooleanArgs,
+      output: '<App key={2} a b />',
+      errors: [expectedError]
+    },
+    {
+      code: '<App b a />',
+      options: reservedFirstAsBooleanArgs,
+      output: '<App a b />',
+      errors: [expectedError]
+    },
+    {
       code: '<App dangerouslySetInnerHTML={{__html: "EPR"}} key={2} b />',
       options: reservedFirstAsBooleanArgs,
+      output: '<App key={2} b dangerouslySetInnerHTML={{__html: "EPR"}} />',
```

Comment:
I think `dangerouslySetInnerHTML` is not a reserved prop name for this component. I think it is only considered reserved for DOM components and not react components. This [code](https://github.com/yannickcr/eslint-plugin-react/blob/master/lib/rules/jsx-sort-props.js#L208) is deleting `dangerouslySetInnerHTML` for the cases where the component is not a DOM component.